### PR TITLE
[MGX-392] Multi use crowdloan rewards

### PR DESCRIPTION
**Pull request by mateuszaaa**

### CODE REVIEW

File: src/weights.rs [hunk 1]
```diff
@@ -64,87 +64,93 @@ pub trait WeightInfo {
...
+			.saturating_add(T::DbWeight::get().reads(6))
+			.saturating_add(T::DbWeight::get().writes(4))
 	}
 }
 
 // For backwards compatibility and tests
 impl WeightInfo for () {
 	fn initialize_reward_vec(x: u32) -> Weight {
-		Weight::from_parts(143_109_000, 0)
+		Weight::from_ref_time(143_109_000)
```
Comment (devdanco) [inline src/weights.rs:110]: Why did you remove from_parts and instead you have from_ref_time ? .. from_ref_time is deprecated

**Review by vanderian — APPROVED** [review]

**Review by devdanco — APPROVED** [review]
